CONTRIBUTING.MD

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
```

Comment:
we can also include issue template if needed or available

Reply:
I agree. Want to add in a separate issue/PR pair?

Reply:
@pombredanne do we have any issue template or do we even need one?

Reply:
we can have one. Simpler is better though, and this has never been an issue for now. So this can wait